### ISSUE TITLE
Case Fan at fixed speed since v207 update

### ISSUE THREAD

**Issue by anak8502** [Rem0o/FanControl.Releases#2805]
Hi
I am running v207 and have a 3 pin case fan that used to be controlled correctly with the previous version. It is now fixed at 100% load. Thankfully it is the NH-S12 ULN so it is running at 850 rpm which isn't too bad.

But recalibrating or repairing the sensors hasn't helped. I also deleted my entire profile and started from scratch but I cannot manually override the speed.

Any ideas? I am using an Asus z170 pro gaming motherboard.
Thank you

**Comment by anak8502** [comment]
Thank you!
That didn't change it, but it seemed the 3-pin fan had an issue in BIOS as it was set to PWM. All fixed now.

Another question: I attached 2 NF-a12 fans to the my MSI 3070 via the mini 4-Pin headers. I cannot get them to run between 1% - 30% ~1.000RPM. I tried changing the 0% hardware curve override but it doesn't change that they start at 1.000rpm. I'd like to set it to 500rpm as the fans are not audible at that speed.

Regards!
